ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -0,0 +1,200 @@
+            Workspace and the consequences thereof.</IndentedListItem>
+          <IndentedListItem>I will know and follow all applicable laws, regulations, and policies regarding access and use
+            for any external data, files, or software that I upload into my Workspace.</IndentedListItem>
+          <IndentedListItem>I will <strong>NOT</strong> upload data or files containing personally identifiable
+            information (PII).</IndentedListItem>
+          <IndentedListItem>PII means information that can be used to distinguish or trace the identity of an individual
+            (e.g., name, social security number, biometric records, etc.) either alone, or when combined with other
+            personal or identifying information that is linked or linkable to a specific individual (2 CFR §
+            200.79)</IndentedListItem>
+          <IndentedListItem>I will use any external data, files, or software that I upload into my Workspace
+            exclusively for the research purpose I have provided for that Workspace.</IndentedListItem>
+          <IndentedListItem>I will <strong>NOT</strong> use any external data, files, or software that I upload into
+            my Workspace for any malicious purposes.</IndentedListItem>
+          <IndentedListItem>If any import of data, files, or software into my Workspace results in unforeseen
+            consequences and/or unintentional violation of these terms, I will notify the <AoUTitle/>
+            as soon as I become aware using the appropriate process.</IndentedListItem>
+        </IndentedUnorderedList>
+        <li>use a version of the <AoUTitle/> database that is current at or after the time my
+          analysis begins.</li>
+        <IndentedUnorderedList>
+          <IndentedListItem>Archive versions of the database are maintained for the sole purpose of completion of
+            existing studies or replication of previous studies. New work may not be initiated on
+            archive versions of the database.
+          </IndentedListItem>
+        </IndentedUnorderedList>
+      </li>
+      <li>share the results of my demonstration project and all contents of my Workbench with the <AoUTitle/>. </li>
+      <li>My workbench and its contents may be made public for the benefit of all authorized users.</li>
+      <li>honor the contribution of those who take part in <i>All of Us</i> to my work
+        <IndentedUnorderedList>
+          <li>I will acknowledge the <AoUTitle/> and its research participants
+            in all oral and written presentations, disclosures, and publications resulting from
+            any analyses of the data.
+            <IndentedUnorderedList>
+              <li>Here is an example acknowledgement statement:</li>
+              <IndentedUnorderedList>
+                <IndentedListItem>
+                  “The <AoUTitle/> is supported by the National Institutes of Health, Office of the
```

Comment:
This does not appear bulleted in the doc (Indented to the level above, but no bullet)